src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -196,7 +196,7 @@
 
 CUSTOMIZATIONS = """
 /* OpenSSL 1.0.2+ verification error codes */
-#if OPENSSL_VERSION_NUMBER >= 0x10002002L && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Probably need to handle this beta2 check separately as well (there's also one on line 215)